go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/terraform-providers/terraform-provider-aws
 go 1.13
 
 require (
-	github.com/aws/aws-sdk-go v1.30.21
+	github.com/aws/aws-sdk-go v1.30.26
```

Comment:
Hi @jayolmos 👋 Quick note that the CI failures for this PR are due to the changes to `go.mod` and `go.sum` -- if you remove those changes this failure should go away. 👍

Reply:
Thanks @bflad ! yeah now it passes the tests but I'm not sure how the provider updates the aws-sdk-go library, this change is only available from 1.30.26.

Reply:
Its automatically handled by Renovate: https://github.com/terraform-providers/terraform-provider-aws/pull/13190 (these typically get reviewed and merged once or twice a week)